src/components/Filters/Filters.types.ts

```diff
@@ -0,0 +1,13 @@
+import { Option } from './Select/Select.types';
+
+export interface FilterRow {
+  uuid: string;
+  conditionOptions: Option[];
```

Comment:
I'm checking with @MateCziner if this has to be this flexible if we can just use some list with posible input value types to determine what conditions should be available.

My original idea was to provide someting like:
```ts
{
  name: string;
  dataPoint: string; // <- table column identifier
  valueType: string | number | datetime | ...; // what can be used as value type
  onRemove: () => void;
  onAdd?: () => void; // we can use it for event tracking
  isFilterApplied?: boolean;
}